projects/packages/status/src/class-modules.php

```diff
@@ -441,7 +442,7 @@ public function activate( $module, $exit = true, $redirect = true ) {
 				}
 			}
 
-			if ( class_exists( 'Jetpack_Plan' ) && ! \Jetpack_Plan::supports( $module ) ) {
+			if ( Jetpack_Plan::supports( $module ) ) {
```

Comment:
Tyop. Seems to be what's making the E2E tests fail.
```suggestion
			if ( ! Jetpack_Plan::supports( $module ) ) {
```

Reply:
Good catch!